sysdump/defaults.go

```diff
@@ -15,6 +16,7 @@ const (
 const (
 	DefaultCiliumLabelSelector               = labelPrefix + "cilium"
 	DefaultCiliumEnvoyLabelSelector          = labelPrefix + "cilium-envoy"
+	DefaultCiliumHelmChartName               = defaults.HelmReleaseName
```

Comment:
https://github.com/liyihuang/cilium-cli/commit/f093eec1c5fee350d8b900d62341b926da4932b8#r123316956


follow the discussion from here


If that's the case, is it better just to follow what has been done like other parameters?

Reply:
no, it's best to use the defaults package here, that's what we expect elsewhere in the CLI for the release name